package.json

```diff
@@ -68,6 +68,7 @@
     "typecheck": "vue-tsc --noEmit && nuxi typecheck playground && nuxi typecheck docs && nuxi typecheck devtools && cd playground-vue && vue-tsc --noEmit",
     "test": "vitest",
     "test:vue": "vitest -c vitest.vue.config.ts",
+    "test:vue:build": "vite build playground-vue",
```

Comment:
Wouldn't it be better to run the actual build command to set it fail? 🤔

Something like `"pnpm --filter 'playground-vue' build"`

Reply:
indeed, ultimately we should probably also run a type checking step but this pr just verifies that the vue app can build